package.json

```diff
@@ -48,9 +48,9 @@
     "pre-commit:lint": "yarn lint-staged",
     "pre-commit:types": "yarn tsc",
     "pre-commit:test": "yarn test",
-    "test": "cross-env CI=1 react-scripts test --env=jsdom",
-    "test:watch": "react-scripts test --env=jsdom",
-    "test:debug": "react-scripts --inspect-brk test --runInBand --no-cache",
+    "test": "cross-env CI=1 react-scripts test --env jest-environment-jsdom-fourteen",
+    "test:watch": "react-scripts test --env jest-environment-jsdom-fourteen",
+    "test:debug": "react-scripts --inspect-brk test --runInBand --no-cache --env jest-environment-jsdom-fourteen",
```

Comment:
When I tried to use `waitFor`, our current tests failed with "TypeError: MutationObserver is not a constructor #731".

I found [this issue documented](https://github.com/testing-library/react-testing-library/issues/731), that suggests using a different jest environment jsdom version.

People in that thread recommended both fourteen and sixteen, and link to a doc recommending fifteen.
sixteen works, so I'm going with that.